src/conditionals/programs_with_decision_points/medium/q6/Gamer.java

```diff
@@ -0,0 +1,34 @@
+package conditionals.programs_with_decision_points.medium.q6;
```

Comment:
Also the checklist on the first page is mainly for you, so feel free to check the boxes off